crates/bindings-csharp/Runtime/build/SpacetimeDB.Runtime.targets

```diff
@@ -9,7 +36,7 @@
   <!-- Adapted from https://github.com/dotnet/dotnet-wasi-sdk/blob/2dbb00c779180873d3ed985e59e431f56404d8da/src/Wasi.Sdk/build/Wasi.Sdk.targets#L245-L262 -->
   <!-- Executes before the errors in https://github.com/dotnet/runtime/blob/57fd56a99d4c97ac2f95fe84640f2a3f653f4dd7/src/mono/wasi/build/WasiApp.Native.targets#L41-L50. -->
   <!-- TODO: remove when https://github.com/dotnet/runtime/issues/82788 is resolved. -->
-  <Target Name="ObtainWasiSdk" BeforeTargets="_SetupWasiSdk">
+  <Target Name="ObtainWasiSdk" BeforeTargets="_SetupWasiSdk;CheckWasmSdks">
```

Comment:
sorry, sanity check - what is this / where does it come from?

Reply:
Nope, this whole thing is pretty hacky as it work arounds bits missing in .NET itself and requires looking at MSBuild internals.